Marlin/Configuration.h

```diff
@@ -1181,14 +1181,14 @@
  *     |    [-]    |
  *     O-- FRONT --+
  */
-#define NOZZLE_TO_PROBE_OFFSET { 10, 10, 0 }
+#define NOZZLE_TO_PROBE_OFFSET { -44.110, -6.170, 0 }
```

Comment:
not specifying Z offset here to prevent crushing the nozzle against the bed. The value of the offset will be calibrated with babystepping